doc/help/elektra-data-structures.md

```diff
@@ -366,3 +366,46 @@ The algorithm uses the `syncbits` to decide if the key set needs to be
 synchronised.
 
 Continue reading [with the error handling](elektra-error-handling.md).
+
+## Order Preserving Minimal Perfect Hash Map
+
+All structs are defined [in](/src/include/opmphm.h).
```

Comment:
The link here will look something like this:

> All structs are defined [in](/src/include/opmphm.h).

I think it would be better if you used the following Markdown code:

```Markdown
All structs are defined in [opmphm.h](/src/include/opmphm.h).
```

This way, the sentence is correct even after we convert the Markdown code:

> All structs are defined in [opmphm.h](/src/include/opmphm.h).